projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -386,7 +392,14 @@ const IssueScreenWithPath = React.memo(
                                 retry()
                             }}
                         />
-                        <IssueScreenHeader issue={issue} />
+                        <IssueScreenHeader
+                            issue={issue}
+                            headerStyles={
+                                specialEditionProps
+                                    ? specialEditionProps.headerStyle
+                                    : undefined
+                            }
+                        />
 
                         <WithBreakpoints>
                             {{
```

Comment:
There are 3 other instances of IssueScreenHeader in the file and they should match the branding